docs/detections/prebuilt-rules/rule-desc-index.asciidoc

```diff
@@ -187,43 +197,63 @@ include::rule-details/disable-windows-firewall-rules-via-netsh.asciidoc[]
 include::rule-details/disabling-user-account-control-via-registry-modification.asciidoc[]
 include::rule-details/disabling-windows-defender-security-settings-via-powershell.asciidoc[]
 include::rule-details/domain-added-to-google-workspace-trusted-domains.asciidoc[]
+include::rule-details/downloaded-shortcut-files.asciidoc[]
+include::rule-details/downloaded-url-files.asciidoc[]
 include::rule-details/dumping-account-hashes-via-built-in-commands.asciidoc[]
 include::rule-details/dumping-of-keychain-content-via-security-command.asciidoc[]
 include::rule-details/dynamic-linker-copy.asciidoc[]
+include::rule-details/esxi-discovery-via-find.asciidoc[]
+include::rule-details/esxi-discovery-via-grep.asciidoc[]
+include::rule-details/esxi-timestomping-using-touch-command.asciidoc[]
 include::rule-details/eggshell-backdoor-execution.asciidoc[]
 include::rule-details/elastic-agent-service-terminated.asciidoc[]
 include::rule-details/emond-rules-creation-or-modification.asciidoc[]
 include::rule-details/enable-host-network-discovery-via-netsh.asciidoc[]
 include::rule-details/encoded-executable-stored-in-the-registry.asciidoc[]
 include::rule-details/encrypting-files-with-winrar-or-7z.asciidoc[]
 include::rule-details/endpoint-security.asciidoc[]
-include::rule-details/enumerating-domain-trusts-via-nltest.exe.asciidoc[]
+include::rule-details/enumerating-domain-trusts-via-dsquery-exe.asciidoc[]
+include::rule-details/enumerating-domain-trusts-via-nltest-exe.asciidoc[]
 include::rule-details/enumeration-command-spawned-via-wmiprvse.asciidoc[]
 include::rule-details/enumeration-of-administrator-accounts.asciidoc[]
 include::rule-details/enumeration-of-kernel-modules.asciidoc[]
+include::rule-details/enumeration-of-kernel-modules-via-proc.asciidoc[]
 include::rule-details/enumeration-of-privileged-local-groups-membership.asciidoc[]
 include::rule-details/enumeration-of-users-or-groups-via-built-in-commands.asciidoc[]
+include::rule-details/exchange-mailbox-export-via-powershell.asciidoc[]
 include::rule-details/executable-file-creation-with-multiple-extensions.asciidoc[]
 include::rule-details/execution-from-unusual-directory-command-line.asciidoc[]
 include::rule-details/execution-of-com-object-via-xwizard.asciidoc[]
 include::rule-details/execution-of-file-written-or-modified-by-microsoft-office.asciidoc[]
 include::rule-details/execution-of-file-written-or-modified-by-pdf-reader.asciidoc[]
 include::rule-details/execution-of-persistent-suspicious-program.asciidoc[]
-include::rule-details/execution-via-electron-child-process-node.js-module.asciidoc[]
-include::rule-details/execution-via-mssql-xp_cmdshell-stored-procedure.asciidoc[]
+include::rule-details/execution-via-electron-child-process-node-js-module.asciidoc[]
+include::rule-details/execution-via-mssql-xp-cmdshell-stored-procedure.asciidoc[]
```

Comment:
Note: @terrancedejesus some of these references were renamed when building via automation